base.ini

```diff
@@ -1,7 +1,7 @@
 [app:app]
 use = egg:snovault#snowflakes
 sqlalchemy.url = postgresql:///snowflakes
-tm.attempts = 3
+retry.attempts = 3
```

Comment:
what is this parameter for?

Reply:
I've only seen it in tests, we have a test making sure it's 3.